app/views/content_items/_print_meta_data.html.erb

```diff
@@ -0,0 +1,33 @@
+<br/>
```

Comment:
I've moved it to `content_items/html_publication/_print_meta_data.html.erb`. Hope that makes it a bit clearer.